refactor: align landing footer

Darguima opened the pull request:
Landing Footer Fixed. Please just note that the right texts of the footer have different behavior about the wrap, that as far as I know, isn't so easy to fix it with just css. Although there isn't any text overflow and the align is fixed.

https://github.com/cesium/seium.org/assets/49988070/2a95004b-4959-4eb5-8966-0e9f64e4472c


Issue: https://github.com/cesium/seium.org/issues/500

ruilopesm reviewed (commented):
I think adding `text-center whitespace-nowrap` will lead to a more consistent text positioning. Try it out and return feedback 💪🏼

Darguima commented:
> I think adding `text-center whitespace-nowrap` will lead to a more consistent text positioning. Try it out and return feedback 💪🏼

The problem is that the space isn't enough to show all the text, so the text-center can't work properly. I will try to better fit the content and give some feedback

![image](https://github.com/cesium/seium.org/assets/49988070/608d0335-7e2d-484e-bd0d-178ff38b3ec3)

Darguima commented:
The text alignment seems be the problem. I changed it to be aligned to the start/left and with the no wrap . Now we don't see the inconsistent on wrap or alignment. 

I changed the position of the links to try create a better alignment. What do you think now?
I think that the only problem now is that the icons at the bottom are aligned to right, and they will be aligned with the bottom right Link, but not with the right top Link


https://github.com/cesium/seium.org/assets/49988070/c3275739-30c2-40a4-9246-9dd6f060619c
